cellxgene_schema_cli/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to the python package `cellxgene-schema` are documented in t
 The format of this changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.1] - 2021-10-06
+
+### Changed
+
+- A performance fix was implemented. Improves the performance of `Validator._is_raw()` in `validate.py`, and addresses timeouts when validating datasets with high number of non-zero values in the expression matrix.
```

Comment:
~A performance fix was implemented.~ Improves the performance of `Validator._is_raw()` in `validate.py` and addresses timeouts when validating datasets with a high number of non-zero values in the expression matrix.